app/invocation/cache_requests_card.tsx

```diff
@@ -698,10 +699,18 @@ export default class CacheRequestsCardComponent extends React.Component<CacheReq
     }
 
     const currentCommand = this.props.model.explicitCommandLine();
-    const cmd1 = commandWithRemoteRunnerFlags(currentCommand + " --experimental_execution_log_compact_file=inv1");
+    let cmd1: string;
```

Comment:
nit: Rename to something like `generateExecLogCmd1` so it's more clear why this is skipped in some cases. For cmd2 as well